WordPressAuthenticator/NUX/NUXButton.swift

```diff
@@ -1,54 +1,157 @@
 import UIKit
 import WordPressShared
+import WordPressUI
 
 /// A stylized button used by Login controllers. It also can display a `UIActivityIndicatorView`.
-@objc open class NUXButton: NUXSubmitButton {
-    // MARK: - Configuration
-    fileprivate let horizontalInset: CGFloat = 20
-    fileprivate let verticalInset: CGFloat = 12
-    fileprivate let maxFontSize: CGFloat = 22
+@objc open class NUXButton: UIButton {
+    @objc var isAnimating: Bool {
+        return activityIndicator.isAnimating
+    }
+
+    @objc let activityIndicator: UIActivityIndicatorView = {
+        let indicator = UIActivityIndicatorView(activityIndicatorStyle: .white)
+        indicator.hidesWhenStopped = true
+        return indicator
+    }()
+
+    override open func layoutSubviews() {
+        super.layoutSubviews()
+
+        if activityIndicator.isAnimating {
+            titleLabel?.frame = CGRect.zero
+
+            var frm = activityIndicator.frame
+            frm.origin.x = (frame.width - frm.width) / 2.0
+            frm.origin.y = (frame.height - frm.height) / 2.0
+            activityIndicator.frame = frm.integral
+        }
```

Comment:
Looks good!! but now that i realize about it: why not just Autolayout? (`centerXAnchor` / `centerYAnchor`)

Reply:
I'm drawing the line here. Need to get this PR closed. :)